src/core/Alert/Alert.md

```diff
@@ -0,0 +1,109 @@
+`<Alert>` is used at the top of a web page to convey information regarding the entire page or web service.
```

Comment:
```suggestion
`<Alert>` is used at the top of a web page to convey information regarding the entire page or web service. It has the ARIA role "alert", which causes any current screen reader output to be interrupted. It should only be used for information that requires the user's immediate attention.
```

The above would be in line with the current functionality. Whether this component _should_ always have the alert role is another question.

Reply:
Alert role was used here because it was read upon page load on most browsers, which we found to be a desired behavior in this case. The component shouldn't be updated dynamically to avoid any problems with it though. It could be discussed if only the error variant should have the alert role.

Reply:
role="alert" (or role="status") shouldn't really be used at all if this component is meant to be loaded along with the rest of the page, in which case it would come up in the regular reading order.

Reply:
Yes. It is meant to be loaded with the rest of the content. The rationale here was that as alert (at least the error variant) is meant to convey any critical issues or problems with the service possibly preventing use, it should be very hard to miss. And here we ended up leaning on the experienced behavior of the alert role.

Reply:
Based on this discussion I would add this sentence to the docs: "It has the ARIA role `'alert'`, which ensures the component's content is read by screen readers immediately on page load."

Reply:
It doesn't quite ensure it since the action varies by browser, so as not to promise too much it might be better to talk about "enhancing screen reader accessibility" or something along those lines.